### ISSUE TITLE
Installation error on OS X 10.11

### ISSUE THREAD

**Issue by drlightx** [michaelhush/M-LOOP#7]
I followed the installation instructions using anaconda, and creating a python environment with scikit-learn (named chipGA). I get an error preventing me from installing M-LOOP:

$ python setup.py test
Traceback (most recent call last):
  File "~/anaconda/envs/chipGA/lib/python3.5/site-packages/setuptols-26.1.1-py3.5.egg/setuptools/dist.py", line 434, in fetch_build_egg
AttributeError: 'Distribution' object has no attribute '_egg_fetcher'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "setup.py", line 41, in <module>
    'Topic :: Scientific/Engineering :: Physics']
  File "~/anaconda/envs/chipGA/lib/python3.5/distutils/core.py", ine 108, in setup
    _setup_distribution = dist = klass(attrs)
  File "~/anaconda/envs/chipGA/lib/python3.5/site-packages/setuptols-26.1.1-py3.5.egg/setuptools/dist.py", line 348, in __init__
  File "~/anaconda/envs/chipGA/lib/python3.5/site-packages/setuptols-26.1.1-py3.5.egg/setuptools/dist.py", line 394, in fetch_build_eggs
  File "~/anaconda/envs/chipGA/lib/python3.5/site-packages/setuptols-26.1.1-py3.5.egg/pkg_resources/__init__.py", line 851, in resolve
  File "~/anaconda/envs/chipGA/lib/python3.5/site-packages/setuptols-26.1.1-py3.5.egg/pkg_resources/__init__.py", line 1123, in best_match
  File "~/anaconda/envs/chipGA/lib/python3.5/site-packages/setuptols-26.1.1-py3.5.egg/pkg_resources/__init__.py", line 1135, in obtain
  File "~/anaconda/envs/chipGA/lib/python3.5/site-packages/setuptols-26.1.1-py3.5.egg/setuptools/dist.py", line 453, in fetch_build_egg
  File "~/anaconda/envs/chipGA/lib/python3.5/site-packages/setuptols-26.1.1-py3.5.egg/setuptools/dist.py", line 418, in get_egg_cache_dir
PermissionError: [Errno 13] Permission denied: './.eggs'

Here's the output of 'conda list' for the environment:

mkl                       11.3.3                        0  
numpy                     1.11.1                   py35_0  
openssl                   1.0.2h                        2  
pip                       8.1.2                    py35_0  
python                    3.5.2                         0  
readline                  6.2                           2  
scikit-learn              0.17.1              np111py35_2  
scipy                     0.18.0              np111py35_0  
setuptools                26.1.1                   py35_0  
sqlite                    3.13.0                        0  
tk                        8.5.18                        0  
wheel                     0.29.0                   py35_0  
xz                        5.2.2                         0  
zlib                      1.2.8                         3

**Comment by drlightx** [comment]
After running the install command with su privileges, it appeared to install properly, but when testing the installation, I get:

...
============================== test session starts ===============================
platform darwin -- Python 3.5.2, pytest-3.0.2, py-1.4.31, pluggy-0.3.1
rootdir: ~/M-LOOP, inifile: 
collected 0 items / 1 errors 

===================================== ERRORS =====================================
____________________ ERROR collecting tests/test_examples.py _____________________
ImportError while importing test module '~/M-LOOP/tests/test_examples.py'.
Original error message:
'No module named 'sklearn.gaussian_process.kernels''
Make sure your test modules/packages have valid Python names.
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!
============================ 1 error in 5.28 seconds =============================

**Comment by drlightx** [comment]
OK, if I ignore Anaconda envs altogether, and install both scikit-learn and M-LOOP manually (using admin privileges), the M-LOOP installation works and passes the test. 

I wonder what Anaconda does when I create an env with scikit-learn that's different from installing manually.

**Comment by michaelhush** [comment]
Great to hear it installed correctly. Requiring users install the dev version of scikit-learn was probably a mistake. I have had trouble on Windows systems with this as well. I'll look into removing that requirement.

**Comment by michaelhush** [comment]
I packaged the code required from scikit-learn in M-LOOP. Now it does not have to be installed before M-LOOP. I've updated the documentation to reflect this. This should stop the first error you got. Instructions on how to update your installation are here http://m-loop.readthedocs.io/en/latest/install.html. 

I am not sure how to what is going on with Anaconda, but as I cannot reproduce the error, and your installation seems to be working now I will close this issue.
